template.adoc

```diff
@@ -58,7 +58,11 @@ include::{common-includes}/gitclone.adoc[]
 // they will be building.
 === Try what you'll build
 
+include::{common-includes}/tryintro.adoc[]

```

Comment:
Looks like the adoc file names in the "include" links need to be updated.

Reply:
Fixed